lib/experimental/workflow-palette.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Enqueues the assets required for the Workflow Palette.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Enqueue workflow palette assets on all admin pages.
+ */
+function gutenberg_enqueue_workflow_palette_assets() {
+	wp_enqueue_script( 'wp-workflow' );
```

Comment:
Do we need to enqueue the script and styles even if the experiment is disabled?

If we don't need maybe we can even remove the global, because if the script loads we know the experiment is enabled.